logging.conf

```diff
@@ -26,12 +26,12 @@ level=DEBUG
 formatter=simpleFormatter
 args=(sys.stdout,)
 
-[handler_fileHandler]
-class=FileHandler
-level=DEBUG
-formatter=simpleFormatter
-args=('robottelo.log', 'a')
+# [handler_fileHandler]
+# class=FileHandler
+# level=DEBUG
+# formatter=simpleFormatter
+# args=('robottelo', 'a')
```

Comment:
Does this change break anything? What's the purpose? I don't see anything in the commit message about it.